src/middlewared/middlewared/plugins/kubernetes_linux/lifecycle.py

```diff
@@ -154,6 +154,12 @@ async def post_start_internal(self):
             except CallError:
                 self.logger.error('Failed to remove %r daemonset', daemonset['metadata']['name'], exc_info=True)
 
+        await self.middleware.call(
+            'k8s.node.remove_taints', [
+                k['key'] for k in (node_config['spec']['taints'] or []) if k['key'] in ('ix-svc-start', 'ix-svc-stop')
+            ]
+        )
+
         # Let helm re-create load balancer services for scaled up apps
```

Comment:
We do need to wait here, for the other,, build-in, taints to be removed as well.

Reply:
Good point @Ornias1993, going to make a ticket for this as it is essentially separate from this and we should likely be not doing a bunch of other things too if built in taints are still active.

Reply:
This worked fine for me btw. Tested live on my machine.

But yea build in taints need to be handled. For now it just left me couple of pods in "error" state, But new ones already have been spun up and working.

Reply:
@sonicaj Agreed, thats also what @stavros-k and I came up with and why we approved it either way :)